src/common/latest/prepare-task.ts

```diff
@@ -20,7 +20,7 @@ export const prepareTask: TaskJob = async (endpointType: EndpointType) => {
   const endpoint = Endpoint.getEndpoint(tl.getInput(endpointType, true) as string, endpointType);
   const rootPath = __dirname;
 
-  const scannerMode = ScannerMode[tl.getInput("scannerMode") as ScannerMode];
+  const scannerMode = ScannerMode[tl.getInput("scannerMode", true)?.toLowerCase() as ScannerMode];
```

Comment:
`toLowerCase()` ensures retro-compatibility with users still using `CLI` / `Other` instead of `cli` / `other`. Users having `scannerMode: MSBuild` will be broken though as they need to move to `dotnet`